frontend/src/metabase/admin/performance/components/StrategyForm.tsx

```diff
@@ -159,11 +165,17 @@ const StrategyFormBody = ({
 
   return (
     <FormWrapper>
-      <StyledForm style={{ ...formStyle }}>
-        <FormBox className="strategy-form-box">
+      <StyledForm style={{ overflow: isInSidebar ? undefined : "auto" }}>
+        <FormBox
+          className="strategy-form-box"
+          style={{
+            paddingInlineStart: isInSidebar ? "2rem" : "2.5rem",
+            paddingInlineEnd: isInSidebar ? "13px" : "2.5rem",
```

Comment:
13px 😱  please push back against such abominations with design

also, this would be much cleaner if this was encapsulated in a styled component, which this already is